go.mod

```diff
@@ -55,7 +58,8 @@ require (
 	github.com/xeipuuv/gojsonschema v1.1.0
 	go.uber.org/fx v1.9.0
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
-	google.golang.org/appengine v1.4.0 // indirect
+	google.golang.org/api v0.10.0 // indirect
+	google.golang.org/appengine v1.5.0
```

Comment:
that wasn't intentional... removing

Reply:
take that back, it was already there

Reply:
something keeps adding that appengine dep, keep forgetting to look into it, oh well